tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -335,3 +335,41 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+  @issue-enterprise-5709
```

Comment:
We don't need this issue tag here. Normally we put issue tags on the scenarios that were written to cover a bug that was fixed for a documentation purpose or a bug that still exists to indicate that this scenario might not pass locally. 
```suggestion

```